main.tex

```diff
@@ -1,4 +1,4 @@
-\documentclass{article}
+ƒtodo\documentclass{article}
```

Comment:
I presume this is a typo when trying to search for the TODO? :)

Reply:
Wooops, you're right! 😅